src/qt/pivx/sendcustomfeedialog.h

```diff
@@ -19,7 +20,7 @@ class SendCustomFeeDialog : public QDialog
     Q_OBJECT
 
 public:
-    explicit SendCustomFeeDialog(QWidget *parent = nullptr);
+    explicit SendCustomFeeDialog(PIVXGUI* _window, QWidget *parent = nullptr);
```

Comment:
No need to add another argument and store it, just change `QWidget *parent` to `PIVXGUI* parent`. The window is being set as parent, it's the same object treated differently.

Reply:
done